src/component/general-help/general-help.vue

```diff
@@ -152,6 +152,13 @@
 
 				<p>{{ $t('weapons_2') }}</p>
 				<p>{{ $t('weapons_3') }}</p>
+				
+				<center>
```

Comment:
`<center></center>` is now deprecated. What about using CSS for centering and replacing non-breakable spaces